nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -231,58 +265,66 @@ Edit `$NAUTOBOT_ROOT/nautobot_config.py`, and head over to the documentation on
+    ### Configuring NAPALM
 
-To use NAPALM, add `nautobot[napalm]` to your `local_requirements.txt` so that it can be installed and kept up to date:
+    Nautobot provides built-in support for the [NAPALM automation](https://github.com/napalm-automation/napalm/) library, which allows Nautobot to fetch live data from devices and return it to a requester via its REST API. The [`NAPALM_USERNAME`](../configuration/optional-settings.md#napalm_username) and [`NAPALM_PASSWORD`](../configuration/optional-settings.md#napalm_password) configuration parameters define the credentials to be used when connecting to a device.
 
-```no-highlight
-echo "nautobot[napalm]" >> $NAUTOBOT_ROOT/local_requirements.txt
-```
+    To use NAPALM, add `nautobot[napalm]` to your `local_requirements.txt` so that it can be installed and kept up to date:
 
-### Remote File Storage
+    ```no-highlight title="Add napalm to the local_requirements.txt file"
+    echo "nautobot[napalm]" >> $NAUTOBOT_ROOT/local_requirements.txt
+    ```
 
-By default, Nautobot will use the local filesystem to store uploaded files. To use a remote filesystem, install the [`django-storages`](https://django-storages.readthedocs.io/en/stable/) library and configure your [desired storage backend](../configuration/optional-settings.md#storage_backend) in `nautobot_config.py`.
+    ### Remote File Storage
 
-To use remote file storage, add `nautobot[remote_storage]` to your `local_requirements.txt` so that it can be installed and kept up to date:
+    By default, Nautobot will use the local filesystem to store uploaded files. To use a remote filesystem, install the [`django-storages`](https://django-storages.readthedocs.io/en/stable/) library and configure your [desired storage backend](../configuration/optional-settings.md#storage_backend) in `nautobot_config.py`.
 
-```no-highlight
-echo "nautobot[remote_storage]" >> $NAUTOBOT_ROOT/local_requirements.txt
-```
+    To use remote file storage, add `nautobot[remote_storage]` to your `local_requirements.txt` so that it can be installed and kept up to date:
+
+    ```no-highlight title="Add remote storage to local_requirements.txt"
+    echo "nautobot[remote_storage]" >> $NAUTOBOT_ROOT/local_requirements.txt
+    ```
 
-An example of using django-storages with AWS S3 buckets, visit the [django-storages with S3](../guides/s3-django-storage.md) user-guide.
+    For an example of using django-storages with AWS S3 buckets, visit the [django-storages with S3](../guides/s3-django-storage.md) user-guide.
 
 ## Prepare the Database
 
 Before Nautobot can run, the database migrations must be performed to prepare the database for use. This will populate the database tables and relationships:
 
-```no-highlight
+```no-highlight title="Push migrations onto the Nautobot database"
 nautobot-server migrate
 ```
```

Comment:
Example output? Or maybe a note explicitly saying that this is verbose and so isn't given as an example here?